src/stories/containers/Finances/FinacesContainer.tsx

```diff
@@ -1,28 +1,79 @@
+import styled from '@emotion/styled';
 import Container from '@ses/components/Container/Container';
 import PageContainer from '@ses/components/Container/PageContainer';
-
+import { useThemeContext } from '@ses/core/context/ThemeContext';
 import React from 'react';
-
+import BreadcrumbYearNavigation from './SeccionPages/BreadcrumbYearNavigation';
 import BreakdownChart from './SeccionPages/BreakdownChart';
 import BreakdownTable from './SeccionPages/BreakdownTable';
-import CardChartOverview from './SeccionPages/CardChartOverview';
-import CardsNavigation from './SeccionPages/CardsNavigation';
+import CardChartOverview from './SeccionPages/CardChartOverview/CardChartOverview';
 import LatestExpenseReports from './SeccionPages/LatestExpenseReports';
 import MakerDAOExpenseMetrics from './SeccionPages/MakerDAOExpenseMetrics';
+import { useFinances } from './useFinances';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
 
-const FinancesContainer = () => (
-  <PageContainer>
-    <Container>
-      <div>MakerDAO Finances </div>
+const FinancesContainer = () => {
+  const { isLight } = useThemeContext();
```

Comment:
Could you move this to the hook in order to have only one "logic" operation here?

Reply:
you are right donde